runtime/src/main/java/com/datastax/oss/quarkus/runtime/graal/NativeSubstitutions.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.quarkus.runtime.graal;
+
+import com.datastax.oss.driver.internal.core.os.Native;
+import com.oracle.svm.core.annotate.Substitute;
+import com.oracle.svm.core.annotate.TargetClass;
+
+@TargetClass(Native.class)
+final class NativeSubstitutions {
```

Comment:
I'm musing a bit here... do we want to include isCurrentTimeMicrosAvailable() here as well (with a constant return value of false) until we get the native stuff sorted out?  We're side-stepping the issue entirely now by forcing the use of a Java clock but my thought was that that's an indirect consequence of not having native support.

On the other hand that clock is the only consumer of this functionality... so I can see a counter-argument which says adding it here is just one more thing we have to unset/change when we want to confirm native functionality is fine.

I guess I could go either way here.

Reply:
We will improve it - see my comments here:
https://datastax-oss.atlassian.net/browse/JAVA-2696
I think that `Substitutions` should be used only if there is no other way to tackle the problem.
For the `isCurrentTimeMicrosAvailable` we have `force_java_clock` setting so it should be ok to use it.